test/integration/legacy_event.ts

```diff
@@ -188,4 +198,24 @@ describe('Event Function', () => {
       assert.deepStrictEqual(receivedContext, test.expectedContext);
     });
   });
+
+  it('returns a 500 if the function throws an exception', async () => {
+    const server = getServer(() => {
+      throw 'I crashed';
+    }, 'event');
+    await supertest(server)
+      .post('/')
+      .send({
+        eventId: 'testEventId',
+        timestamp: 'testTimestamp',
+        eventType: 'testEventType',
+        resource: 'testResource',
+        data: {some: 'payload'},
+      })
+      .set({'Content-Type': 'application/json'})
+      .expect(res =>
+        assert.deepStrictEqual(res.headers['x-google-status'], 'error')
+      )
+      .expect(500);
```

Comment:
Would be good to check that the exception message doesn't appear in the response body.

Reply:
done